uid/snowid_test.go

```diff
@@ -301,8 +301,6 @@ func FuzzID_Parse_RoundTrip_FromString(f *testing.F) {
 	testCases := []string{
 		"self",
 		"abcdefghi",
-		"123456789",
-		"1",
```

Comment:
These cases will be found pretty quickly again when running in fuzz mode. You can try it out with `-fuzz FuzzID_Parse_RoundTrip_FromString`.

I think we need to keep these, but handle the normalization in the test logic below with something like this at the end:

```go
normalized := strings.TrimLeft(original, "1")
assert.Equal(t, id.String(), normalized, "int64=%d", id)
```